modules/drivers/googleanalytics/test/metabase/driver/googleanalytics_test.clj

```diff
@@ -243,12 +243,11 @@
                                                (t/local-date "2019-11-18")
                                                (t/local-time 0)
                                                (t/zone-id system-timezone-id)))
-                                  (t/zone-id system-timezone-id))
+                                   (t/zone-id system-timezone-id))
         (is (= expected-ga-query
                (do-with-some-fields
-                (fn [{:keys [db event-action-field event-label-field date-field], :as objects}]
+                (fn [{:keys [db], :as objects}]
                   (qp.store/with-metadata-provider (u/the-id db)
-                    (qp.store/bulk-metadata :metadata/column (map u/the-id [event-action-field event-label-field date-field]))
```

Comment:
Why is this removed instead of inlined?

Reply:
it was only done for side effects, the old QP store code (before we switched the QP to MLv2 metadata providers) would error if you tried to access something that wasn't already fetched. The new MetadataProviders just fetch it instead of complaining. This line is left over from the PR that switched everything to MetadataProviders, we're not getting any benefit from it except maybe saving one app DB call in a test